learners/demo_lessons.md

```diff
@@ -66,7 +66,7 @@ Use the links below to jump to the starting points information for a particular
 |---------- |--------------- |--------------- |------------------------- |
 | [Data Organization in Spreadsheets](https://datacarpentry.org/spreadsheet-ecology-lesson/index.html) | \* [Dates as Data](https://datacarpentry.org/spreadsheet-ecology-lesson/03-dates-as-data.html)<br/> \* [Quality Control](https://datacarpentry.org/spreadsheet-ecology-lesson/04-quality-control.html) | \* [Introduction](https://datacarpentry.org/spreadsheet-ecology-lesson/00-intro.html)<br/> \* [Formatting Data Tables in Spreadsheets](https://datacarpentry.org/spreadsheet-ecology-lesson/01-format-data.html)<br/> \* [Formatting problems](https://datacarpentry.org/spreadsheet-ecology-lesson/02-common-mistakes.html)<br/> \* [Exporting data](https://datacarpentry.org/spreadsheet-ecology-lesson/05-exporting-data.html) |  |
 | [Data Cleaning with OpenRefine](https://datacarpentry.org/OpenRefine-ecology-lesson/index.html) |  \* [Exploring Data with OpenRefine](https://datacarpentry.org/OpenRefine-ecology-lesson/03-exploring-data.html) | \* [Introduction](https://datacarpentry.org/OpenRefine-ecology-lesson/01-getting-started.html) | The other episodes depend on actions taken earlier in the lesson - consult [_Teaching Episodes with Dependencies_](#dependencies). |
-| [Data Analysis and Visualization in R](https://datacarpentry.org/R-ecology-lesson/index.html) |  \* [Introduction to R](https://datacarpentry.org/R-ecology-lesson/01-intro-to-r.html)<br/>\* [Starting with Data](https://datacarpentry.org/R-ecology-lesson/02-starting-with-data.html) | \* [Before We Start](https://www.datacarpentry.org/R-ecology-lesson/00-before-we-start.html) | [Manipulating Data](https://datacarpentry.org/R-ecology-lesson/03-dplyr.html) and [Data Visualization with ggplot2](https://datacarpentry.org/R-ecology-lesson/04-visualization-ggplot2.html) depend on actions taken earlier in the lesson - consult [_Teaching Episodes with Dependencies_](#dependencies). |
+| [Data Analysis and Visualization in R](https://datacarpentry.org/R-ecology-lesson/index.html) |  \* [Data Visualization with ggplot2]https://datacarpentry.org/R-ecology-lesson-alternative/visualizing-ggplot.html) | \* [Introduction to R and RStudio](https://datacarpentry.org/R-ecology-lesson-alternative/introduction-r-rstudio.html) | The other episodes depend on actions taken earlier in the lesson - consult [_Teaching Episodes with Dependencies_](#dependencies). |
```

Comment:
```suggestion
| [Data Analysis and Visualization in R](https://datacarpentry.org/R-ecology-lesson-alternative/index.html) |  \* [Data Visualization with ggplot2]https://datacarpentry.org/R-ecology-lesson-alternative/visualizing-ggplot.html) | \* [Introduction to R and RStudio](https://datacarpentry.org/R-ecology-lesson-alternative/introduction-r-rstudio.html) | The other episodes depend on actions taken earlier in the lesson - consult [_Teaching Episodes with Dependencies_](#dependencies). |
```

Reply:
Hmm, I have used the wrong URL here altogether: we should be linking to `R-ecology-lesson` rather than `*-alternative` at this point.

Reply:
addressed in ce99df1